packages/next-swc/crates/core/src/lib.rs

```diff
@@ -99,7 +103,15 @@ pub fn custom_before_pass(
     cm: Arc<SourceMap>,
     file: Arc<SourceFile>,
     opts: &TransformOptions,
-) -> impl Fold {
+) -> impl Fold + '_ {
+    let relay_plugin = {
```

Comment:
Same here, `#[cfg]` for `wasm` is required.

Reply:
Done: https://github.com/vercel/next.js/pull/33839